backend/api/src/create-market.ts

```diff
@@ -155,6 +157,10 @@ export async function createMarketHelper(body: Body, auth: AuthedUser) {
     : unmodifiedAnte
   const ante = Math.min(unmodifiedAnte, totalMarketCost)
 
+  if (await isDuplicateSubmission(idempotencyKey, pg)) {
+    throw new APIError(400, 'Contract has already been created')
```

Comment:
Would be great to return the url for them to visit, but I can add this later if that's too much trouble.

Reply:
Will do!